addons/gmc-toolbox/show_creator_dock.gd

```diff
@@ -92,6 +123,61 @@ func _ready():
 # 		self.config.set_value("lights", l, settings)
 # 	self.config.save(CONFIG_PATH)
 
+func _load_lights():
+
+	self.lights = []
+	self.tags = []
+	var scene = load(edit_show_scene.text).instantiate()
+	# Look for a lights child node
+	var lights_node = scene.get_node_or_null("lights")
+	if not lights_node:
+		debug_log("No lights node in show scene")
+		return
+
+	for child_light in lights_node.get_children():
+		if child_light is GMCLight:
+			self.lights.append(child_light)
+			for tag in child_light.tags:
+				if not self.tags.has(tag):
+					self.tags.append(tag)
+
+	for child in tags_container.get_children():
+		child.queue_free()
```

Comment:
You should remove the child from the container first, before queue free, to avoid dead pointers.